src/mahoji/commands/easter.ts

```diff
@@ -13,21 +37,67 @@ export const easterCommand: OSBMahojiCommand = {
 		description: 'Allows you to do the 2022 Easter Event.'
 	},
 	options: [],
-	run: async ({ channelID, userID }) => {
+	run: async ({ userID }) => {
 		const user = await client.fetchUser(userID.toString());
 		if (user.minionIsBusy) return `${user.minionName} is busy.`;
+		const eggsGivenOut = user.settings.get(UserSettings.EggsDelivered);
+		const cl = user.cl();
+		if (cl.has(UniqueTable.allItems)) {
+			return "I don't have anything left to give you!";
+		}
+
+		if (eggsGivenOut > 0) {
+			// Remove all their eggs, give a loot roll for each one they've delivered.
+			await mahojiUserSettingsUpdate(client, user.id, {
+				eggs_delivered: {
+					decrement: eggsGivenOut
+				}
+			});
+			let loot = DeliverRewardTable.roll(eggsGivenOut);
+
+			// Can't get more than one of each unique.
+			for (const item of UniqueTable.allItems) {
+				if (!loot.has(item)) continue;
+				if (cl.has(item)) {
+					loot.remove(item, loot.amount(item));
+				} else {
+					// Make sure they're not getting more than 1 at once.
+					loot.bank[item] = 1;
+				}
+			}
+
+			await user.addItemsToBank({ items: loot, collectionLog: true });
+
+			return {
+				content: `You received ${loot}.`,
+				attachments: [
+					{
+						fileName: 'image.jpg',
+						buffer: await newChatHeadImage({
+							content: 'Thank you for delivering eggs, please take these as a reward!',
```

Comment:
Maybe tell how many eggs they turned in, like `Thank you for delivering ${eggsGivenOut} eggs, take your reward and keep up the good work!`